frontend/src/exporter/ExporterLogin.tsx

```diff
@@ -0,0 +1,157 @@
+            },
+        ],
+        // This is separate from the login form, so that the form can be submitted even if a general error is present
+        generalError: [
+            null as { code: string; detail: string } | null,
+            {
+                setGeneralError: (_, error) => error,
+                clearGeneralError: () => null,
+            },
+        ],
+    }),
+    forms(({ actions }) => ({
+        login: {
+            defaults: { password: '' } as LoginForm,
+            errors: ({ password }) => ({
+                password: !password ? 'Please enter your password to continue' : undefined,
+            }),
+            submit: async ({ password }, breakpoint) => {
+                breakpoint()
+                const response = await fetch(window.location.href, {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify({ password }),
+                })
+                if (response.status == 200) {
+                    const data = await response.json()
+                    actions.setData(data)
+                } else {
+                    actions.setGeneralError(response.statusText, (await response.json()).error)
+                }
+            },
+        },
+    })),
+])
+
+export interface ExporterLoginProps {
+    whitelabel?: boolean
+}
+
+export function ExporterLogin(props: ExporterLoginProps): JSX.Element {
+    const { data, isLoginSubmitting, generalError } = useValues(loginLogic())
+
+    if (data) {
+        return <Exporter {...data} />
+    }
+
+    const login = (
+        <div className="space-y-4">
+            <h2>Access share</h2>
+            {generalError && (
+                <LemonBanner type="error">
+                    {generalError.detail || ERROR_MESSAGES[generalError.code] || (
+                        <>
+                            Could not complete your login.
+                            <br />
+                            Please try again.
+                        </>
+                    )}
+                </LemonBanner>
+            )}
+            <Form logic={loginLogic} formKey="login" enableFormOnSubmit className="space-y-4">
+                <div className={clsx('PasswordWrapper')}>
+                    <LemonField
+                        name="password"
+                        label={
+                            <div className="flex flex-1 items-center justify-between gap-2">
+                                <span>Password</span>
+                            </div>
+                        }
+                    >
+                        <LemonInput
+                            type="password"
+                            className="ph-ignore-input"
+                            data-attr="password"
+                            placeholder="••••••••••"
+                            autoComplete="current-password"
```

Comment:
This is likely going to trigger people's password managers which is not what we want for this kind of dashboard?